encoding/v2/bench/benchmark_eigenda_test.go

```diff
@@ -212,28 +212,52 @@ func benchmarkMultiproofGeneration(b *testing.B, encodingConfig encoding.Config)
 			rsExtendedBlobPowerBytes := blobPowerBytes + 3
 			rsExtendedBlobPowerFrs := rsExtendedBlobPowerBytes - 5 // 32 bytes per Fr element
 			rsExtendedBlobFrs := uint64(1) << rsExtendedBlobPowerFrs
+			blobFrs := uint64(1) << (blobPowerBytes - 5) // original blob size in field elements
 			params := encoding.EncodingParams{
 				NumChunks:   8192,                           // blob_version=0
 				ChunkLength: max(1, rsExtendedBlobFrs/8192), // chosen such that numChunks*ChunkLength=rsExtendedBlobFrs
 			}
 			provingParams := prover.ProvingParams{
-				BlobLength:  1 << (blobPowerBytes - 5),
+				BlobLength:  blobFrs,
 				ChunkLength: max(1, rsExtendedBlobFrs/8192), // chosen such that numChunks*ChunkLength=rsExtendedBlobFrs
 			}
 			parametrizedProver, err := p.GetKzgProver(params, provingParams)
 			require.NoError(b, err)
 
 			for b.Loop() {
-				_, err = parametrizedProver.GetProofs(b.Context(), maxSizeBlobCoeffs[:rsExtendedBlobFrs])
+				_, err = parametrizedProver.GetProofs(b.Context(), maxSizeBlobCoeffs[:blobFrs])
```

Comment:
hmm why this change? We want to be generating proofs for the rs extended blob lengths. Otherwise I have no idea how the code above works. why is chunkLength being set to `rsExtendedBlobFrs/8192` for example.

Feel like you changed this place but its probably the proving params that we wanted to change instead?

Reply:
ah! My bad then... I guess it was always like this.. I hallucinated some stuff here. Oh probably because before it was calling GetFrames and forgot to change after I refactored to only call GetProofs?

Anyways sgtm. Wondering if theres a cleaner way to parametrize `chunkLength` though that doesn't require defining all those extendedBlobLen variables